### ISSUE TITLE
Knowers Profile

### ISSUE THREAD

**Issue by Nathael** [blue-horizon-lab/connecty-ico#36]
The knowers profile is a kind of resume, with additional features, formated to Connecty defined format in order to allow searchs within the knowers database.
It contains the following parts (to be completed) :
- (human) languages (spoken / read), with for each :
   * native speaker / learnt speaker
   * personnal estimation of skill
   * buyers rated skill
- studies/education (university, masters, ....) (Get official list of universities and schools)
- knowledges / skills, with, for each :
   * number of years of practice
   * personnal estimation of skill
   * buyers rated skill
   * peer rated skill (votes limited to CONNECTY validated experts in the corresponding domain)
- experiences outside of connecty
- experiences within connecty, with
   * duration (start, end, length)
   * personnal evaluation
   * buyers evaluation
   * list of skills involved

It may also contain additional information like :
- Availability (10%, 50%, 100%)
- Average daily rate for dev / support / advice / ... (possibly depending 
- Kind of missions accepted (short / long)
- ...

**Comment by slamazelo** [comment]
Remarque commune aux autres profiles : 
**Il faudrait préciser ici ce qui est du domaine du front ou du back, sinon on risque de développer deux fois la même chose (sauf pour la bdd evidemment)** 
Après discussion avec Olivier, on part en première étape sur un profile simplifié en attendant de savoir si cela ressort du back ou du front, et à quel moment il faudra développer le full profile. 
En première estimation, les specs ci-dessus supposent beaucoup plus de travail que çà en a l'air, et il faudra plus de précisions sur les évaluations/ratings, gestion des listes de skills...

**Comment by Nathael** [comment]
Tout ce qui est listé ici relève du front
Il s'agit d'un Epic, que je découperais en stories, comme les autres.
Il faut vraiment que l'on s'appelle pour que lon utilise l'outil (zenhub) de la même façon, et que l'on ait la même vision de ce qui est à faire :)

**Comment by slamazelo** [comment]
oui, je veux bien, mais il faut aussi préciser dans le stories, please !
OK on sort de cet epic
